docs/whatsnew/1.0.rst

```diff
@@ -9,6 +9,24 @@ What's New in Astropy 1.0?
 Overview
 --------
 
+Astropy 1.0 is a major release that adds significant new functionality since the 0.4.x series of releases.
+
+In particular, coordinate conversions to/from Altitude/Azimuth are now
+supported (see `Support for Alt/Az coordinates`_), a new package to help with
+data visualization has been added (see :ref:`whatsnew_viz`), and a new package
+for common analytic functions is now also included (see
+:ref:`whatsnew_analytical_functions`).
+
+The :mod:`astropy.io.ascii` sub-pacakge now includes fast C-based
+readers/writers for common formats, and also supports a new ASCII format the
+better preserves meta-data (see :ref:`whatsnew_io_ascii`), the modeling package
+has been significantly improved and now supports composite models (see
+:ref:`whatsnew_modeling`), and the :class:`~astropy.table.Table` class can now
+include :class:`~astropy.coordinates.SkyCoord` and :class:`~astropy.time.Time`
+objects containing arrays (see :ref:`whatsnew_table`).
+
+In addition to these major changes, Astropy 1.0 includes a large number of
+smaller improvements, which are described in the `Full change log`_
```

Comment:
Maybe mention that there are bug fixes unique to this release as well. (Just "smaller improvements *and* bug fixes")